### ISSUE TITLE
Can't add RT.com rss feed to Brave news dashboard.

### ISSUE THREAD

**Issue by HGhostRevisionist88** [brave/brave-browser#21422]
Hi,

There is a problem adding feeds from https://www.rt.com/rss-feeds/ like https://www.rt.com/rss-feed/ to brave, none of the rss feeds in the list work, It says: "Sorry, we couldn't find a feed at that address.".
I had no problems when I added the ones from RT Spanish https://actualidad.rt.com/feeds/all.rss and I didn't have problems adding others like Sputnik English and Spanish to Brave, they worked perfectly fine.

I am using: Brave	1.38.16 Chromium: 99.0.4844.51 (Official Build) nightly (64-bit) 
Revision	d537ec02474b5afe23684e7963d538896c63ac77-refs/branch-heads/4844@{#875}

And also tested this issue in latest Release v1.36.109 (Chromium 99.0.4844.51) and same error.

Another test I made was to download the rss file from RT and loaded to Brave with a local server and it was added without issue, it worked just perfectly fine, minus the fact that it won't get updated. But it was added without errors, so, what could it be the problem then if the rss is fine apparently?

I hope you can fix it whatever the issue is, I don't think it is a server issue and I only saw that "couldn't find a feed" when I didn't type the whole url with https:// and all, so I don't understand why Brave is not finding any feed in the link.

Thank you and have a good day.

**Comment by HGhostRevisionist88** [comment]
Seems like: http://tass.com/rss/v2.xml also gives the same error "Sorry, we couldn't find a feed at that address." but I tested adding: https://tass.ru/rss/v2.xml which is the Russian version and it loaded with no problems at all.

Seems suspicious but hope it is just a small little bug.

I also tested other RT languages and [Russian](https://russian.rt.com/rss), [Arabic](https://arabic.rt.com/rss/), [German](https://de.rt.com/feeds/news/) and all English rss failed to be added, the only ones that worked were [French](https://francais.rt.com/rss) and [Spanish ](https://actualidad.rt.com/feeds/all.rss)were added and worked fine.

**Comment by HGhostRevisionist88** [comment]
Ok nevermind, you are obviously doing something clownish about it, I find it suspicious how you literally removed RT from News Sources https://github.com/brave/news-aggregator/pull/9 and magically today it doesn't work even if I want to manually add it myself?

Does @BrendanEich or @bbondy (although he co-founded Brave, I really don't know if he thinks or express the same as Brendan) even know what you are doing? is he even aware or it? He is always talking about censorship and how big tech shouldn't tell you what to search, read, see or not.
You haven't implemented goggles in Brave Search, you still use Google search to poison the already weird results in Brave search.

Where is what Brendan Eich promised? in forgotten or nevermind land? You are literally doing what Microsoft and Google are doing, can you trust them? I am not even asking to add RT, I literally noticed you REMOVED it 4 days ago and somehow now I can't manually add RT English, that looks suspicious, very suspicious. 

There was literally this issue: https://github.com/brave/brave-browser/issues/19357 where someone literally decided to change some stupid wording and label it "inclusive" because GOOGLE guidelines say it, are you even aware of that? The problem is not the use of stupid inclusive BS, the problem is Google says everyone jumps and apparently the supposedly anti-big tech Brave, still jumped.

I could make another issue in the news-aggregator questioning WHY RT was removed in the first place, I kind of was confused when I opened News yesterday trying to test to add some rss news feed, and then I questioned myself "wasn't RT available already"?
But I said, nah, maybe not, why would Brave remove it, Brendan told me that Brave is not like Google or Microsoft.

You might agree or disagree with RT news, but you are literally doing what Twitter is doing and Youtube is doing, and even worst... because in Twitter, you use Nitter server with an IP from USA or whatever, and you can bypass their "withheld" censorship, you can really workaround those issues even without VPN. But Brave removed RT, that means, nobody will be able to read it, and I can't even manually add the RSS because magically it doesn't work, but the subdomain "actualidad" which is the Spanish version does work.

Why did you even remove RT from news sources? besides the fact that you look like another puppet from the big tech? give me a reasonable and real answer why anti-big tech browser did it. You will say because it is financed by Russian Government? is that it? then it is MY choice to see if I want to read it or not and get it on my news.
What about mainstream media news outlets? Tell me, who are financing them? because all I have seen in the past days from those super 'non-RussianGOV financed sources' how they have used from videogames, fake imagines with people with fake guns, videos from years ago from the whole Donbass conflict shelling being passed as "in 2022 war... Russia doing it", explosions and conflicts from not even Ukraine or Russia, it is fake after fake after fake after fake... and have you removed those sources? no.
At least RT has actually reported on the Donbass issues which obviously you wouldn't know if you watched CNN because a 8 year old conflict that is being caused by 3rd parties to destabilize region is 1. nothing new and 2. it can be ignored by many who just look for the 'mainstream media narrative'.

So those are my reasons WHY all your excuses why you removed RT will be just BS, which means Brave is literally becoming what Brendan Eich complained about at some point and either Brendan agrees or he is not even aware of that. 

This is all BS, and I might just find an alternative browser, because I wasn't already too happy with the whole Wallet update you did, it was literally "new = wallet", I don't use Brave rewards or anything so I don't care about it. But I was ignoring that, but I won't ignore this type of BS.

You can just be honest and admit you don't care and you are not here to really fight Big Tech BS, you are here for money and in the money business your ideals and real dreams won't matter much.
But don't play the "we are different" when you have shown, time after time that you do what you are told, directly or indirectly, like I have shown you with the whole "inclusive language" issue, because how terrible is if you don't do what Google, the king of the internet says.

I am almost done and it sucks because I have been using Brave for many years, installing it on family and friends computers and phones, but Brave has shown to be just the same as Microsoft and Google, and at this point only manifest v2 vs v3 is the only thing that makes me think what Brave is supposedly offering over most of the other browsers, but then now I seriously doubt the integrity of Brave team promises, because like I say, time after time I have seen how you have done things and it just seems like not far from the Big tech you supposedly complain about, not different than Google or Microsoft or Mozilla and their whole pro censorship and de-platforming views.

Have a nice day.

**Comment by HGhostRevisionist88** [comment]
@jonathansampson

It works now, yesterday when I created the issue, it wasn't working, so I created the issue because I didn't know if it was someone doing something with the server or just Brave not getting the feed fine or something. That's why I even tested downloading the RSS and sideloading it, to make sure it wasn't something about the xml or whatever and then Brave rejecting it, since I could access the rss in Brave and no 404 error or anything like a server problems, I decided to create the ticket. Of course, with all the crap that is going on, anyone can mess with anything and then magically something doesn't work.

However, my post today, this long post as you can see has almost nothing to do with the original issue, it was linked but still not the same, yes, now I can manually add the rss in Brave now, but why was it removed in the first place? I never used Brave news because it didn't have news I would read, now I can add my own I can actually use Brave news, that's why I was adding them.
But then I was wondering to myself "wasn't RT available? nah, maybe not" and then I searched for RT and found out someone removed it 4 days ago. 

so, why? why do I have to manually add rss (which wasn't even working yesterday), when it was available before in the list of news feeds to toggle on and off? I always have my suspicious about everything, I won't trust anyone, so I wasn't sure if it was Brave or not messing with it, so I asked. But when I saw you literally made me manually add the rss when it was available 4 days ago, it's just weird, especially when Brendan and Brave always talk about anti censorship and anti big tech BS decisions.
Is Brave really doing it? Is it really going against Big tech? because as you can see someone in Brave did the whole "let's use inclusive language" because Google guidelines, not Brave guidelines, which would have been the same waste of time, but at least it meant that you were the ones doing it, and not being influenced by Google.

Thank you for your answer, it seems like it works now so whatever it was, governments messing with it, maybe when my router restarted today it changed IP and that fixed it, I wish someone would have replied sooner so I didn't search and witness how Brave removed RT, somehow proving that they are not full 'anti-censorship' like I once thought.

Have a nice day Mr. Sampson, I tested adding RT feed before reading your message, so it wasn't your reply the reason I tried again and added it, but it is cool you responded to tell me if it was working for you or not and it could be my side that for whatever reason decided not to accept English Russians news outlets. So I thank you for that, since it was the original issue.
But again, it doesn't explain why so much anti big tech talk and someone in your team removed RT.

So I am closing this issue since it is resolved because it is magically working now.

**Comment by jonathansampson** [comment]
Manually adding RSS feeds _was working yesterday_ too. That said, our team was able to reproduce the issue when attempting to load those feeds on our end as well—however the behavior was not consistent (what you would observe with an endpoint under immense load, or protected by anti-DDoS services, etc.).

**Brave does not censor.** That said, we also make no guarantee that _default lists_ will include particular properties and sources; those default lists have changed in the past, and will very likely change in the future from time to time.

We believe it is up to the user where they wish to source their news, which is why we indiscriminately support the inclusion of custom feed sources. We won't get in your way 🙂

All the best, Ghost! Please do let me know if there is ever anything further we can do for you.

**Comment by HGhostRevisionist88** [comment]
I noticed RT runs anti-DDoS for a week (for reasons we know), so that might be a reason for the error, and today I am getting 502 errors in RT, so I guess anything could be the reason from the problem yesterday, but I have added like 15 sources and only had problems with the RT and TASS. 

And believe me, I try to believe Brave doesn't censor, But I read Google and Microsoft were going to censor RT and Sputnik and other news outlets that don't say what western mainstream media says and then today I see Brave removed RT 4 days ago, so it was suspicious, if it was 2 weeks ago it would have been okaish like a funny coincidence, but doing that exactly when the big tech kinds of the internet are doing it seemed like it was done for strange decisions. So I can't be blamed for my suspicion, you know, never trust anyone too much.

But I will have to trust your word, because I have read you in Twitter and see you as an honest person who cares about Brave, I will believe that RT got removed because life and coincidence in a middle of the conflict and not because of some person in the Brave team trying to do some **ignorant activism**, like what MS, Apple, Google are doing etc, or the ignorant people pulling vodkas from their bars or _russian_ cats banned from participating in international competitions, and removing anything that has the name 'Russian', sports, soccer, world cup and etc etc.
In the end, at least now I can add them myself, but I won't lie when I hate to see RT just got removed while garbage mainstream fake media outlets still there, in the world where propaganda vs propaganda exists, I guess RT lost.

Thank you for your answer and have a great day @jonathansampson take care.
